openproblems/tasks/dimensional_reduction/README.md

```diff
@@ -1,11 +1,15 @@
-### Dimensional reduction manifold preservation
----
+## Dimensional Reduction
 
-Dimensional reduction is one of the key challenges in single-cell data representation. Routine single-cell RNA sequencing (scRNA-seq) experiments measure cells in roughly 20,000-30,000 dimensions (i.e., features - mostly gene transcripts but also other functional elements encoded in mRNA such as lncRNAs.) Since its inception, scRNA-seq experiments have been growing in terms of the number of cells measured. Originally, cutting-edge SmartSeq experiments would yield <a href="#">a few hundred cells</a>, at best. Now, it is not uncommon to see experiments that yield over <a href="https://www.nature.com/articles/s41586-018-0590-4">100,000 cells</a> or even <a href="https://www.10xgenomics.com/blog/our-13-million-single-cell-dataset-is-ready-to-download">> 1 million cells</a>.
+Dimensional reduction is a key challenge in single-cell data representation.
```

Comment:
Could we cite a paper or two on the challenges here?

Reply:
Thanks for suggestion, adding.